buildkite/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  - &common_plugins
+    - &docker_plugin
+      docker#v3.8.0:
+        image: &ruby_version "ruby:3.2.2"
+
+steps:
+  #################
+  # Build and Test
+  #################
+  - group: "🧪 Build and Test"
+    key: test
+    steps:
+      - label: "🧪 Build and Test"
+        command: |
+          echo "--- :rubygems: Setting up Gems"
+          bundle install
+
+          echo "--- :rspec: Run Rspec"
+          bundle exec rspec --profile 10 --format progress
+        plugins: *common_plugins
+        agents:
+          queue: "default"
```

Comment:
This queue setting is redundant: The `default` queue is used if no `agents.queue` is set.

Reply:
Tbh, personally I like that we make the agent queue explicit in the pipeline file, also to show that this was a conscious choice (†). I'd be ok to set that `agents: queue: "default"` at the top-level of the `pipeline.yml` file to avoid repeating it on each step though and clean things up in the individual steps. WDYT?

---
_(†) especially saying this because [I'm personally experimenting with some stuff in Tumblr right now, trying to make some of our lint steps run on Linux agents instead of Macs if possible](https://github.tumblr.net/TumblrMobile/orangina/pull/25461), and I had to try different agents and think carefully about which one would make more sense for each step)_

Reply:
> I'd be ok to set that agents: queue: "default" at the top-level of the pipeline.yml file to avoid repeating it on each step

Sounds like a good idea 👍

Reply:
> I like that we make the agent queue explicit in the pipeline file, also to show that this was a conscious choice
I agree. I moved the default queue parameter to the top.